index.html

```diff
@@ -510,15 +503,12 @@ <h3>Use Cases and Requirements</h3>
 <a>Issuers</a> can issue revocable <a>verifiable credentials</a>.
           </li>
           <li>
-The processes of cryptographically protecting <a>credentials</a> and
-<a>presentations</a>, and verifying <a>verifiable credentials</a> and
-<a>verifiable presentations</a>, have to be deterministic, bi-directional, and
-lossless. Any verification of a <a>verifiable credential</a> or
-<a>verifiable presentation</a> has to be transformable to the generic data model
-defined in this document in a deterministic process, such that the resulting
-<a>credential</a> or <a>presentation</a> is semantically and syntactically
-equivalent to the original construct, so that it can be processed in an
-interoperable fashion.
+The processes of cryptographically protecting and verifying
+<a>verifiable credentials</a> and <a>verifiable presentations</a>, have to be
+deterministic, bi-directional, and lossless. Any verification of a
+<a>verifiable credential</a> or <a>verifiable presentation</a> has to be
+transformable to the generic data model defined in this document in a
+deterministic process so that it can be processed in an interoperable fashion.
```

Comment:
```suggestion
<a>verifiable credentials</a> and <a>verifiable presentations</a> have to be
deterministic, bi-directional, and lossless. Any
<a>verifiable credential</a> or <a>verifiable presentation</a> has to be
transformable to the generic data model defined in this document in a
deterministic process so that its verification can be processed in an
interoperable fashion.
```